backend/src/main/resources/application.properties

```diff
@@ -12,5 +12,11 @@ spring.h2.console.enabled=true
 spring.h2.console.path=/h2-console
 spring.h2.console.settings.web-allow-others=true
 
+spring.servlet.multipart.enabled=true
+spring.servlet.multipart.max-file-size=5MB
+spring.servlet.multipart.max-request-size=10MB
+
 jwt.secret=changemeeeeeeeeeeeeeeeeeeeeeeeee
 jwt.expiration=3600000
+
+storage.path=./fileStorage
```

Comment:
maye used the /data/ folder as a parent, i.e. `./data/files` where the db is also located